src/pudl/metadata/classes.py

```diff
@@ -927,7 +931,6 @@ def add_datastore_metadata(self) -> None:
             partitions["years"] = partitions["year"]
         elif "year_month" in partitions:
             partitions["year_month"] = max(partitions["year_month"])
-        self.source_file_dict["source_years"] = self.get_temporal_coverage(partitions)
```

Comment:
Is this function getting called elsewhere? If not, why get rid of it here? Is this the change referenced in this line of your PR, "Remove duplicative and flaky source vs. working partitions in data source template. Just show temporal coverage"?

Reply:
It's called in several places, including the Jinja templates for the Datasette Metadata and the individual data source documentation pages.

I removed it here because it was in effect being called twice -- once to generate these source years, and again to generate the "working years" which used to be referenced separately (since there are some years for which data is available, where we haven't extracted that data) but when we went to using the templates, both of them were set to reference the working years. So the same range showed up more than once on the data pages, and also were always called "years" even though we now have quarters, months, and years.